slack-Makefile

```diff
@@ -2,4 +2,7 @@ all:
 	true
 
 install:
-	cp -ra usr/* /app
+	install apply_extra /app/bin
+	install slack.sh /app/bin/slack
+	cp /usr/bin/ar /app/bin
+	cp /usr/lib/libbfd-*.so /app/lib
```

Comment:
Not really sure, copied from the spotify app example. I think `ar` links against libbfd so that's why the copy is needed. But not sure why `ar` needs to be in `/app` in the first place. Maybe `apply_extra` which calls it needs it there. Or there was something weird in spotify that meant `ar` needed to be copied and I'm just duplicating shit I don't understand...